app/src/main/java/com/pizzapp/OrderSummary.java

```diff
@@ -59,8 +60,12 @@ public void setToolbar() {
     }
 
     public boolean onOptionsItemSelected(MenuItem item) {
-        Intent myIntent = new Intent(getApplicationContext(), MainActivity.class);
-        startActivityForResult(myIntent, 0);
+//        Intent myIntent = new Intent(getApplicationContext(), MainActivity.class);
+//        startActivityForResult(myIntent, 0);
+//        return true;
```

Comment:
can these lines be deleted

Reply:
I have deleted these in the second commit, I also don't see these in my final version and it is labeled outdated.